packages/suite/src/views/dashboard/PromoBanner.tsx

```
@@ -5,16 +5,16 @@ import styled, { css } from 'styled-components';
 import { Button, Column, Icon, Image, Row, Tooltip, variables } from '@trezor/components';
 import { isWeb } from '@trezor/env-utils';
 import { EventType, analytics } from '@trezor/suite-analytics';
-import { spacings } from '@trezor/theme';
+import { Button, Icon, Image, Tooltip, variables, Row, Column } from '@trezor/components';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Button'. Consider to delete it or rename it.

'Button' is defined here:


(lint/suspicious/noRedeclare)